app-conf/GeneralConf.xml

```diff
@@ -45,4 +45,14 @@
     <name>drelephant.analysis.fetch.initial.windowMillis</name>
     <value>3600000</value>
   </property> -->
+  <property>
+    <name>drelephant.analysis.retention.period</name>
+    <value>0</value>
+    <description>Number of days of analysis to be kept in the database</description>
+  </property>
+  <property>
+    <name>drelephant.analysis.purge.operation.interval</name>
+    <value>60</value>
```

Comment:
I think once a day or once hour

Reply:
I remove it from the configuration file shipped with the project. Default value will be every day.